app/config/config.yml

```diff
@@ -68,7 +68,11 @@ swiftmailer:
 
 # MassiveSearch Configuration
 massive_search:
+    adapter: elastic
     adapters:
+        elastic:
+            hosts: '%elasticsearch.hosts%'
```

Comment:
demo should work without any problems and as normally a demo is installed on a dev PC without a cluster only  a single host should be in the parameters.yml so we can preconfigure `127.0.0.1:9200` in the paramters.yml.dist file:

```yml
            hosts:
                - "%elasticsearch_host%"
```